src/Faker/Provider/fa_IR/DateTime.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Faker\Provider\fa_IR;
+
+class DateTime extends \Faker\Provider\DateTime
+{
+    /**
+     * Get a date string between January 1, 1970 and now
+     *
+     * @param string                 $format
+     * @param \DateTime|int|string   $max    maximum timestamp used as random end limit, default to "now"
+     * @param  boolean $convertChars False (by default) to keep Digits in normal english format, true to convert to Farsi chars(۰,۱,۲,۳,۴,۵,۶,۷,۸,۹)
+     * @return string
+     * @example '1367-09-28'
+     */
+    public static function date($format = 'Y-m-d', $max = 'now', $convertChars = false)
+    {
+        return JalaliDateTime::date($format, static::dateTime($max)->getTimestamp(), $convertChars);
```

Comment:
Ah, sorry, I just saw it down there. Please add a `use` statement to make the dependency visible.